Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/MapElement.cs

```diff
@@ -103,6 +103,12 @@ public bool IsEnabled
         /// <returns><see cref="MapElement"/></returns>
         public static MapElement FromMapElement(windows.UI.Xaml.Controls.Maps.MapElement args)
```

Comment:
There are more kinds of MapElement, like MapBillboard, MapElement3D, MapPolygon, and MapPolyline.

Why special-case MapIcon only?  Seems like it should be all or nothing?